runtime/westend/src/lib.rs

```diff
@@ -1174,7 +1174,10 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	(pallet_staking::migrations::v10::MigrateToV10<Runtime>,),
+	(
+		pallet_staking::migrations::v10::MigrateToV10<Runtime>,
+		pallet_nomination_pools::migration::v2::MigrateToV2<Runtime>,
+	),
```

Comment:
```suggestion
	(),
```

These don't need to be here anymore

Reply:
ah yeah I was just testing.